### PR TITLE
test possible k8s test fix

### PR DESCRIPTION
**Pull request by ben-toogood**

### CODE REVIEW

File: test/kind.go [hunk 1]
```diff
@@ -81,17 +81,30 @@ func (s *testK8sServer) Run() error {
...
 	}
 
-	ChangeNamespace(s.Command(), s.Env(), s.Env())
-	if s.opts.Login {
-		Login(s, s.t, s.Env(), "micro")
+	// login with this new account
+	Login(s, s.t, s.Env(), "micro")
+
+	// delete the admin account
+	outp, err := s.Command().Exec("auth", "delete", "account", "admin")
```
Comment (domwong) [inline test/kind.go:105]: I thought an account is global? e.g. there is only 1 account called `admin` regardless of namespace.

Comment (ben-toogood) [reply test/kind.go:105]: No, accounts are scoped per namespace.

File: service/auth/server/auth/auth.go [hunk 2]
```diff
@@ -121,7 +121,7 @@ func (a *Auth) Generate(ctx context.Context, req *pb.GenerateRequest, rsp *pb.Ge
 		req.Options = &pb.Options{}
 	}
 	if len(req.Options.Namespace) == 0 {
-		req.Options.Namespace = namespace.DefaultNamespace
+		req.Options.Namespace = namespace.FromContext(ctx)
```
Comment (domwong) [inline service/auth/server/auth/auth.go:124]: What changed such that we needed to make this fix? Why wasn't it picked up in previous tests?

Comment (ben-toogood) [reply service/auth/server/auth/auth.go:124]: This isn't needed for this PR but it's more standard with the other handlers.

**Pull request closed without merging**